common/src/main/scala/explore/common/ConstraintSetObsQueries.scala

```diff
@@ -28,10 +25,23 @@ import lucuma.ui.reusability._
 import monocle.Getter
 import monocle.macros.Lenses
 
+import ConstraintSetObsQueriesGQL._
+
 object ConstraintSetObsQueries {
+
   type ConstraintSetList = KeyedIndexedList[ConstraintSet.Id, ConstraintsSummary]
   type ObsList           = KeyedIndexedList[Observation.Id, ObsSummaryWithPointingAndConstraints]
 
+  sealed trait A
+  object A {
+    case class B(b: Int) extends A
+    case class C(c: String) extends A
+  }
+  implicit val reuseA: japgolly.scalajs.react.Reusability[A] = {
+    import japgolly.scalajs.react._
+    japgolly.scalajs.react.Reusability.derive[A]
+  }
+
```

Comment:
Is this needed?

Reply:
Good catch, thank you. No, that's not needed, I was just testing the `Reusability.derive` under different circumstances. (If `ConstraintSetObsQueries` is a trait instead of an object, it doesn't work).